### DISCUSSION TITLE
PSD Thumbnail not working with 8.5.5.24 or 9.0 beta2

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1424] [General]
Hello,
thumbnail of .psd is not showed but under ImageGlass application is showed.

I have made ImageGlass default application, i have deleted thumbail cache of Wondows 10 Pro or Windows 11 Pro 64 without solution
I have tested on ten computers.

To solve Thumbnail preview i have installe third party codec (FastPictureViewer codec pack)
Have a solution ?
Thanks

**Comment by d2phap** [comment]
ImageGlass does not provide thumbnails for Explorer, so you should use the other software (like https://sourceforge.net/projects/sagethumbs/) for it.

**Comment by donatopace** [comment]
Very good,
but thumbnail generated by "SageThumbs" respect Color Management ICC?
Thanks for suggestion!


Il mer 14 set 2022, 17:03 Phap Dieu Duong ***@***.***> ha
scritto:

**Reply by d2phap** [reply]
You need to give it a try, I am not sure.
Or you can search for another software as well.
